src/main/java/org/dependencytrack/tasks/BomUploadProcessingTask.java

```diff
@@ -327,6 +327,12 @@ by accident, or the default rollback behavior in case of an exception has change
             if (!vulnAnalysisEvents.isEmpty()) {
                 qm.createVulnerabilityScan(TargetType.PROJECT, ctx.project.getUuid(), ctx.uploadToken.toString(), vulnAnalysisEvents.size());
                 vulnAnalysisEvents.forEach(kafkaEventDispatcher::dispatchAsync);
+                // Initiate vuln-analysis workflow for the token
+                var vulnAnalysisState = qm.getWorkflowStateByTokenAndStep(ctx.uploadToken, WorkflowStep.VULN_ANALYSIS);
+                if (vulnAnalysisState != null) {
+                    vulnAnalysisState.setStartedAt(Date.from(Instant.now()));
+                    qm.persist(vulnAnalysisState);
+                }
             }
```

Comment:
and the children tasks will be NOT_APPLICABLE as well?

Reply:
Oh, good point! 

I think in that case we should dispatch policy evaluation and metrics update from here instead, rather than cancelling / skipping them. Them being skipped is even a gap in the current implementation in `main`...